pages/linux/yum.md

```diff
@@ -1,29 +1,7 @@
 # yum
 
-> Package management utility for RHEL, Fedora, and CentOS (for older versions).
-> For equivalent commands in other package managers, see <https://wiki.archlinux.org/title/Pacman/Rosetta>.
-> More information: <https://manned.org/yum>.
+> This command is an alias of `dnf`.
```

Comment:
```suggestion
> This command is an alias of `dnf` on modern distros.
> For older distros (e.g. CentOS 7), `yum` is a distinct program from `dnf`. Some subcommands and options may be different.
```

I thought I'd add this as a courtesy to RHEL7 ELS users (all five of them 😆). Plus, many people are still running CentOS 7 to this day (mostly due to outdated tutorials targeting newbies) even though they ***really*** shouldn't.